docs/interact/cli.mdx

```diff
@@ -3,37 +3,74 @@ title: Using the NEAR CLI
 sidebar_position: 2
 ---
 
-Calimero works with all the NEAR tooling including the NEAR CLI.
+Calimero works with all the NEAR tooling including the NEAR CLI. 
 
 You can install the NEAR CLI using: 
+
 ```
 npm install -g near-cli
 ```
 
+You can always check your current version by running:
+
+```
+near --version  # version info appears on the last line of output
+```
+
+:::info
 All methods from NEAR CLI are compatible with Calimero Private Shards and the documentation can be found on the official NEAR docs: https://docs.near.org/tools/near-cli.
+:::
 
 
-To use NEAR CLI with Calimero SDK you will need to:
+## Requirements
+
+To use NEAR CLI with Calimero you will need to:
 - generate an Authentication token on the Security page inside the Calimero Console
 - set the custom RPC endpoint for the CLI tool to use from your Dashboard
 - set the API token for the CLI to use
 
 
-In case you are using a mainnet private shard, set the `NEAR_CLI_MAINNET_RPC_SERVER_URL` environment variable. On the other hand, for testnet use the
-`NEAR_CLI_TESTNET_RPC_SERVER_URL` environment variable instead.
+## Set API key
```

Comment:
Add info that the `https://rpc.testnet.near.org` value is currently ignored by the CLI, the issue is fixed and will be released in the next version of the CLI.

We have this entire section already written in the console auth token side.
You can reuse it, or better yet, move it from there to here, and link to this content there (since CLI usage should be described here and not under console/auth-token)

Reply:
okay